src/Core/Checkout/Customer/CustomerException.php

```diff
@@ -65,6 +68,8 @@ class CustomerException extends HttpException
     public const VALUE_NOT_SUPPORTED = 'CONTENT__RULE_VALUE_NOT_SUPPORTED';
     public const MISSING_REQUEST_PARAMETER_CODE = 'CONTENT__MISSING_REQUEST_PARAMETER_CODE';
     public const MISSING_OPTIONS = 'CONTENT__MISSING_OPTIONS';
+    public const UNEXPECTED_TYPE = 'CONTENT__UNEXPECTED_TYPE';
```

Comment:
I have not seen this in the other PR, why are you using "CONTENT" as prefix for the codes here? should be checkout